src/pipecat/services/tavus.py

```diff
@@ -0,0 +1,138 @@
+#
+# Copyright (c) 2024, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+"""This module implements Tavus as a sink transport layer"""
+
+# import asyncio
+# from typing import Any
+import base64
+import requests
```

Comment:
We already have use `aiohttp`, any usage of `requests` was removed a while back. See other comment.